src/lib/dev-environment/dev-environment-core.js

```diff
@@ -440,20 +439,16 @@ export async function importMediaPath( slug: string, filePath: string ) {
  *   - If there is a newer version of the WordPress version currently used
  *   - A choice to use a different image
  *
- * @param  {Object=} instancePath Path to local profile
+ * @param  {Object=} slug slug
  * @param  {Object=} options options
  */
-async function updateWordPressImage( instancePath, options ) {
+async function updateWordPressImage( slug, options ) {
 	const versions = await getVersionList();
 	const refRgx = new RegExp( /\d+\.\d+(?:\.\d+)?/ );
-	const vsnRgx = new RegExp( /\$wp_version \= \'(.*)\'\;/ );
-	const landoFile = `${ instancePath }/.lando.yml`;
-	const versionFile = `${ instancePath }/${ DEV_ENVIRONMENT_WORDPRESS_VERSION_FILE }`;
+	const instancePath = getEnvironmentPath( slug );
 
-	// If versionFile doesn't exist it means that the environment has not been initiated.
-	if ( ! fs.existsSync( versionFile ) ) {
-		return;
-	}
+	// Get the current image tag that the WP image is currently using in the .lando.yml file
```

Comment:
The comment here needs updating or? We no longer use `.lando.yml` file, correct?